### PR TITLE
upgraded to babel 6

### PR DESCRIPTION
**Pull request by danderu**
This PR contains the following improvements:

- Simplified test boilerplate, without using the old utilities scripts and not pushing you to use any shallow render solution (until we find the most suitable one).
- Upgraded to Babel 6
- Static references in package.json

@carlosvillu @nucliweb @davecarter please review

### CODE REVIEW

**Comment by carlosvillu** [comment]
+1

File: app/templates/test/_test.js [hunk 1]
```diff
@@ -1,28 +1,13 @@
-import expect from 'expect';
+/* eslint no-unused-expressions:0 */
```
Comment (davecarter) [inline app/templates/test/_test.js]: Just wondering if this linting rule toggling is going to be copied in default test of generated files.

**Comment by davecarter** [comment]
:+1:

File: app/templates/test/_test.js (hunk 1 above)
Comment (danderu) [inline app/templates/test/_test.js]: Yes, it is. Otherwise, the generated component won't pass the linting rules.

**Pull request merged**